services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java

```diff
@@ -38,8 +38,8 @@
     port = Optional.ofNullable(System.getenv("SERVICES_GJ_PORT"));
     BASE_URI = PROTOCOL + host.orElse("localhost") + ":" + port.orElse("80") + "/";
     LSTN_URI = PROTOCOL + lstn.orElse("0.0.0.0") + ":" + port.orElse("80");
-    //String uname = "steve";
-    //String password = "blue";
+    String uname = "steve";
+    String password = "blue";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'password' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://167.99.129.70:9000/coding_rules#rule_key=squid%3AS2068)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "password" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://167.99.129.70:9000/coding_rules#rule_key=squid%3AS1481)